solana/programs/example-native-token-transfers/src/instructions/release_inbound.rs

```diff
@@ -135,25 +173,19 @@ pub fn release_inbound_unlock(
         }
     }
 
-    assert!(inbox_item.release_status == ReleaseStatus::Released);
```

Comment:
Note to self/other reviews: 
Deleting this is OK because `try_release()` ensures that a return value of `true` implies `ReleaseStatus::Released` and vice-versa. 
The `if !released` check above is therefore logically equivalent to this `assert` and so keeping it here is redundant.